### DISCUSSION TITLE
svgMeltedModules.php example only working on dev-main

### DISCUSSION THREAD

**Discussion by yeftaz** [chillerlan/php-qrcode#266] [Q&A]
@YeftaAW how so? Please open [a new discussion](https://github.com/chillerlan/php-qrcode/discussions) for your issue.

_Originally posted by @codemasher in https://github.com/chillerlan/php-qrcode/issues/127#issuecomment-2126340767_
            
I have tried this example: *examples/svgMeltedModules.php* and it's only working on *dev-main* when I was tested it on v5.0-beta, v5.0.0 and above including dev-main.

I just copied and pasted exactly from the example above. Here is the result (same result btw) of those trials which failed.

![732SUyea8UQnNr4c](https://github.com/chillerlan/php-qrcode/assets/71545409/aa3f5808-e773-4c4f-a3ef-22437148749c)

**Comment by codemasher** [comment]
There's a line in bold over here in the readme https://github.com/chillerlan/php-qrcode#documentation and i think that might be the issue. The examples are not compatible between major versions.

**Reply by yeftaz** [reply]
I see, my bad, didn't see that one. Thanks you!
